Product/Production/Services/PatientDiscoveryCore/src/test/java/gov/hhs/fha/nhinc/patientdiscovery/outbound/deferred/request/StandardOutboundPatientDiscoveryDeferredRequestTest.java

```diff
@@ -271,7 +272,7 @@ public void policyFailed() throws ConnectionManagerException {
         verify(pd201305Processor).storeLocalMapping(requestArgument.capture());
         assertEquals(request, requestArgument.getValue().getPRPAIN201305UV02());
 
-        verify(correlationDao).saveOrUpdate("urn:uuid:messageId", patientId);
```

Comment:
For an Outbound request, if assertion.messageId value is present, we don't format it to have urn:uuid: prefix. The concerned test-case reflects this scenario.